packages/hosted-widget-integration/src/HostedWidgetPaymentComponent.tsx

```diff
@@ -141,7 +144,7 @@ class HostedWidgetPaymentComponent extends Component<
         prevState: Readonly<HostedWidgetPaymentMethodState>,
     ): Promise<void> {
         const {
-            deinitializePayment = noop,
```

Comment:
Removed unnecessary `noop` because `deinitializePayment` is a required prop.
So the code coverage got improved.